setup.py

```diff
@@ -42,6 +42,7 @@
         "onnx",
         "onnxruntime<1.15.0",
         "transformers>=4.33.0",
+        "intel-extension-for-pytorch",
```

Comment:
why is it needed ? shouldn't it be optional ?

Reply:
Yes, I move it to be optional, This change is only for automated weekly testing.